utils.h

```diff
@@ -226,10 +226,31 @@ inline std::string size_attr_str(const Token& t, const std::string& prefix = "")
     return count_attr_str(t, prefix);
 }
 
+inline std::string pcount(Decl* p, const std::string& prefix = "")
+{
+    if (p->attrs_ && (p->attrs_->string_ || p->attrs_->wstring_))
+        return prefix + p->name_ + "_len";
```

Comment:
For `wstring` the size is len * sizeof(wchar_t)

Reply:
`pcount` only returns the size of the count (`len`). `psize` will return `sizeof(wchar_t)`